pandas/tests/tslibs/test_conversion.py

```diff
@@ -66,3 +66,12 @@ def test_length_zero_copy(dtype, copy):
     arr = np.array([], dtype=dtype)
     result = conversion.ensure_datetime64ns(arr, copy=copy)
     assert result.base is (None if copy else arr)
+
+
+def test_tz_convert_freq():
+    import pandas as pd
+    import pytz
+    t1 = pd.Timestamp('2019-01-01 10:00', freq='H')
+    assert t1.tz_localize(pytz.utc).freq == t1.freq
```

Comment:
Just assert the `Timestamp`s are equal. IIRC the freq will be compared as well.

Reply:
but i assume `t1.tz_localize(pytz.utc)` is `tz-aware` while `t1` is `tz-naive` ? @mroeschke

Reply:
I guess as mentioned below. I was incorrect in assuming the freq would be compared as well.